webapp/certification/views.py

```diff
@@ -12,4 +12,46 @@
 
 
 def certification_home():
-    return flask.render_template("certification/index.html")
+
+    # Desktop section
+    desktop_releases = []
+    desktop_vendors = []
+
+    for release in api.certified_releases(limit="0")["objects"]:
+        if int(release["desktops"]) > 0 or int(release["laptops"]) > 0:
+            desktop_releases.append(release)
+
+    for vendor in api.certified_makes(limit="0")["objects"]:
+        if int(vendor["desktops"]) > 0 or int(vendor["laptops"]) > 0:
+            desktop_vendors.append(vendor)
+
+    # Server section
+    server_releases = {}
+    server_vendors = api.vendor_summaries_server()["vendors"]
+
+    for index, vendor in enumerate(server_vendors):
+        if index == 0:
+            server_releases = vendor
+            del server_releases["releases"]
+        for version in vendor:
+            if version != "releases":
+                server_releases[version] += vendor[version]
+
+    # IoT section
+    iot_releases = api.certified_releases(smart_core__gte="1")["objects"]
+    iot_vendors = api.certified_makes(smart_core__gte="1")["objects"]
+
+    # SoC section
+    soc_releases = api.certified_releases(soc__gte="1")["objects"]
+    soc_vendors = api.certified_makes(soc__gte="1")["objects"]
+
+    return flask.render_template(
+        "certification/index.html",
+        desktop_releases=desktop_releases,
+        desktop_vendors=desktop_vendors,
+        server_releases=server_releases,
```

Comment:
not passing down `server_vendors`

Reply:
I just actually removed that, because it's not required in the design/wireframe

![image](https://user-images.githubusercontent.com/14939793/112348317-9fa58880-8cbf-11eb-9993-0f406ad5ac0f.png)

Reply:
Ah ok. It's still on the template.

Reply:
Renamed, because I think it's confusing, this is actually raw data from the API, not an empty array like the others.

Reply:
Removed from the template. I think beth is already aware of this anyway.